oberon/drivers/oberon_ecdsa.c

```diff
@@ -477,7 +477,7 @@ psa_status_t oberon_ecdsa_verify_message(
     psa_key_type_t type = psa_get_key_type(attributes);
 
     switch (type) {
-#if defined(PSA_NEED_OBERON_PURE_EDDSA_TWISTED_EDWARDS_255) || defined(PSA_NEED_OBERON_PURE_EDDSA_TWISTED_EDWARDS_448)
+#if (defined(PSA_NEED_OBERON_PURE_EDDSA_TWISTED_EDWARDS_255) || defined(PSA_NEED_OBERON_PURE_EDDSA_TWISTED_EDWARDS_448)) && !defined(PSA_NEED_OBERON_ED25519PH)
```

Comment:
I don't think we can have this extra ifdef here.
We must still be able to run EdDSA pure if PSA_NEED_OBERON_ED25519PH is enabled.

Reply:
Yes true, the config being enabled doesn't necessarily mean it's ED25519ph